openfecwebapp/templates/macros/filters/range.html

```diff
@@ -3,23 +3,20 @@
   <legend class="label">{{ label }}</legend>
   <div class="range">
     <div class="js-filter">
-      <label for="min_{{name}}">From</label>
-      <div class="combo combo--filter--mini">
-        <input id="min_{{name}}" class="combo__input" type="text" name="min_{{name}}" data-inputmask="'alias': 'currency', 'rightAlign': false, 'clearMaskOnLostFocus': true" data-suffix="or more" tabindex="0">
-        <button class="combo__button button--go button--standard is-disabled" type="button">
-          <span class="u-visually-hidden">Search</span>
-        </button>
+      <div class="range__input range__input--min">
```

Comment:
What about just adding the `.js-filter` class to this div? I don't think we need the extra one, and then `range__input` can be the class for styles.